core/data/src/main/java/uk/gov/gchq/gaffer/data/elementdefinition/view/View.java

```diff
@@ -160,6 +162,22 @@ public void expandGlobalDefinitions() {
         }
     }
 
+    @JsonIgnore
+    public Function<Element, Set<Object>> getGroupByFunction() {
```

Comment:
Perhaps change the name of the method to createGroupByFunction so user's are more aware that it is doing some work and creating a new instance of the function each time.